ansible_collections/arista/avd/molecule/evpn_underlay_isis_overlay_ibgp/intended/configs/DC1-SVC3A.cfg

```diff
@@ -89,6 +92,7 @@ interface Ethernet4
    no switchport
    ip address 172.31.255.31/31
    isis enable EVPN_UNDERLAY
+   isis bfd
```

Comment:
Since this device is an MLAG device, I would also expect to find isis and isis bfd on the MLAG peering vlan (4094 in this case)

Reply:
Updated the code to implement this.